### PR TITLE
Unite root path and health check

### PR DESCRIPTION
**Pull request by romanovskyj**
As root path should be the same as health check, `--healthcheck-path` option is deleted

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 setup(
     name='apsconnectcli',
     author='Ingram Micro',
-    version='1.7.0',
```
Comment (hayorov) [inline setup.py]: This is back-compatibility breaking change. You need to bump MAJOR version 1.8.0.

File: README.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ Config saved [/Users/allexx/.aps_config]
 
 ```
 apsconnect install-backend --name NAME --image IMAGE --config-file CONFIG_FILE --hostname HOST \
-                          [--healthcheck-path HEALTHCHECK_PATH] [--root-path ROOT_PATH] \
```
Comment (spukst3r) [inline README.md]: You should leave `--healthcheck-path` argument working, but log a WARNING when it's used. Something like

WARNING --healthcheck-path is deprecated and will be dropped in future releases. The connector should have healthcheck path equal to root path.

File: setup.py (hunk 1 above)
Comment (spukst3r) [inline setup.py]: As long as `--healthcheck-path` is working, we can keep the MAJOR version same. When we drop it, we'll bump it.

File: apsconnectcli/apsconnect.py [hunk 3]
```diff
@@ -175,10 +175,16 @@ def version(self):
         print("apsconnect-cli v.{} built with love."
               .format(pkg_resources.get_distribution('apsconnectcli').version))
 
-    def install_backend(self, name, image, config_file, hostname, healthcheck_path='/',
+    def install_backend(self, name, image, config_file, hostname, healthcheck_path=False,
```
Comment (spukst3r) [inline apsconnectcli/apsconnect.py]: Why `False`? Use `None`.

Comment (spukst3r) [inline apsconnectcli/apsconnect.py]: Why `False`? Use `None`.

Comment (romanovskyj) [inline apsconnectcli/apsconnect.py]: Done

**Pull request merged**